dev/how-to/migrate/from-mariadb-galera-cluster.md

```diff
@@ -0,0 +1,571 @@
+---
+title: Migrate Data from MariaDB Galera Cluster to TiDB
+summary: Learn to migrate data from MariaDB Galera Cluster to TiDB.
+category: how-to
+---
+
+# Migrate Data from MariaDB Galera Cluster to TiDB
+
+This document introduces how to migrate data from MariaDB Galera Cluster to TiDB. 
+
+## Hardware and system requirements
+
+### Hardware requirements
+
+- NIC bonding or teaming
+- PCIe SSDs (not SATA SSDs)
+- CPU power saving management disabled in BIOS
+- NUMA disabled in BIOS (`flush_caches` is optimized in Percona Server 5.7.10. For details, see [Improved NUMA support](https://www.percona.com/doc/percona-server/5.7/performance/innodb_numa_support.html))
+
+### Operating system requirements
+
+- Use CentOS 6.9 64-bit or CentOS 7.3-1611 64-bit as the operating system.
+
+- If you use a virtual machine, do not deploy a cluster on a single physical machine.
+
+- Use XFS as the file system.
+
+- Disable SWAP space by setting `vm.swappiness` to 0.
+
+- If the memory size exceeds 64 GB, enable huge pages.
+
+- If the memory exceeds 32 GB, disable NUMA in the operating system.
+
+- Disable IPv6.
+
+    ```
+    # vim /etc/sysctl.d/99-sysctl.conf
+    net.ipv6.conf.all.disable_ipv6 = 1
+    net.ipv6.conf.default.disable_ipv6 = 1
+    ```
+
+- Enable port forwarding.
+
+    ```
+    # vim /etc/sysctl.d/99-sysctl.conf
```

Comment:
This edits the same file as the previous step. To make things easier for users, we should combine steps when possible.